admin/services/data-access/group.data.service.js

```diff
@@ -11,7 +11,57 @@ const groupDataService = {}
 groupDataService.fetchGroups = async function (query) {
   const q = query || {}
   q.isDeleted = false
-  return Group.find(q).sort({ name: 'asc' }).lean().exec()
+  return Group
+    .find(q)
+    .collation({ locale: 'en', strength: 2 })
```

Comment:
this will likely fail in cosmos :(

Reply:
Arghh...! It's just to make it case insensitive. I'll look into other ways to sort this out.

Reply:
@pris54 speak to george, he had this issue too

Reply:
@pris54 this will definitely fail in Cosmos - just remove the collation and add a HACK or TODO comment and we will sort it with SQL Server